packages/ui/src/theme/overwrites/components/numberInput.ts

```diff
@@ -0,0 +1,38 @@
+export const NumberInput = {
+  sizes: {
```

Comment:
🤔 Doesn't NumberInput sizes match mockups by default? In other words, is there a need to customize them?

Reply:
I checked it and it's match by height but not by padding so I delete height from this customize sizes.